tests/fixtures/base-js-v1-addon/tests/dummy/app/styles/app.css

```diff
@@ -0,0 +1 @@
+/* Ember supports plain CSS out of the box. More info: https://cli.emberjs.com/release/advanced-use/stylesheets/ */
```

Comment:
This was needed, as when copying over an empty file that would fail the stylelint that a freshly generated test-app would be set up for. This is taken from the blueprint: https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/app/styles/app.css